src/tests/java/picard/illumina/parser/CycleIlluminaFileMapTest.java

```diff
@@ -75,42 +75,4 @@ private static String constructPathString(int lane, int cycle) {
             },
         };
     }
-
-  /*  @Test(dataProvider = "iteratorTestData")
```

Comment:
@jacarey , can you comment on why you commented these tests out a couple of years ago?